### PR TITLE
Jpk/keyserials

### PR DESCRIPTION
**Pull request by jpknoll**

### CODE REVIEW

File: Keas.Core/Models/ExpiringItemsEmailModel.cs [hunk 1]
```diff
@@ -1,8 +1,5 @@
 ﻿using System;
-using System.Collections.Generic;
```
Comment (srkirkland) [inline Keas.Core/Models/ExpiringItemsEmailModel.cs]: @jpknoll this makes the build fail for me

**Comment by srkirkland** [comment]
@jpknoll just from running it, the main problem I see is when you are adding a serial from the person page.  It fails but I'm not sure what it should do even if it succeeds.  I think there is a missing step about what key to associate the serial with.

Also, I think most keys won't have names and it fails if you try to add a key w/o a name right now

File: Keas.Mvc/Program.cs [hunk 2]
```diff
@@ -18,7 +12,14 @@ public class Program
         public static void Main(string[] args)
         {
             var host = BuildWebHost(args);
-            
+
+            using (var scope = host.Services.CreateScope())
+            {
+                var dbContext = scope.ServiceProvider.GetRequiredService<ApplicationDbContext>();
+                var dbInitilizer = new DbInitializer(dbContext);
+                dbInitilizer.Initialize();
```
Comment (srkirkland) [inline Keas.Mvc/Program.cs]: @jpknoll danger, danger, danger

Comment (jpknoll) [reply Keas.Mvc/Program.cs]: Go read the init function

Comment (srkirkland) [reply Keas.Mvc/Program.cs]: not so dangerous apparently

**Pull request merged**